packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1216,6 +1242,9 @@ export type ChatAdapterUiState = {
     error?: Error;
 };
 
+// @public
+export type ChatAttachmentType = 'file' | 'inlineImage' | 'unknown';
```

Comment:
Should the 'file' type be exposed to the public API surface?

Reply:
If `file` is still in beta lets no expose the enum yet